### PR TITLE
move channel name change

### PR DESCRIPTION
**Pull request by wanda-carlson**

### CODE REVIEW

**Comment by wanda-carlson** [comment]
comment

**Comment by wanda-carlson** [comment]
Cool stuff! 
Screen Shot 2022-01-11 at 3 40 53 PM

**Comment by henry-richard** [comment]
Good stuff

**Comment by henry-richard** [comment]
Hi wanda

**Comment by henry-richard** [comment]
Hi wanda

**Comment by henry-richard** [comment]
Hello Wandy

**Comment by henry-richard** [comment]
Henry is here now

**Comment by henry-richard** [comment]
Wanda wandaaa

**Comment by henry-richard** [comment]
wanda 2

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # public
 
 This is a public repository managed by PaidPaper. The code is based in Python. It has several GitHub actions.
+
```
Comment (wanda-carlson) [inline README.md:4]: this is weird

Comment (wanda-carlson) [inline README.md:4]: Wanda1

Comment (wanda-carlson) [inline README.md:4]: Zainab's comment

Comment (wanda-carlson) [reply README.md:4]: Another Zainab comment

**Comment by wanda-carlson** [comment]
Hi

**Comment by wanda-carlson** [comment]
Hello

**Comment by wanda-carlson** [comment]
Here's a picture
[Screen Shot 2022-01-06 at 7.59.26 AM.png](https://files.slack.com/files-pri/T0289BY1FK4-F0316B8EJGH/screen_shot_2022-01-06_at_7.59.26_am.png)

**Comment by wanda-carlson** [comment]
Screen Shot 2022-01-10 at 10 25 58 AM

**Comment by wanda-carlson** [comment]
Not a picture
[README.md](https://github.com/PaidPaper/public/files/7965333/README.md)

**Comment by wanda-carlson** [comment]
Not picture
[README.md](https://github.com/PaidPaper/public/files/7965334/README.md)

**Comment by wanda-carlson** [comment]
image attachment
Screen Shot 2022-01-24 at 9 16 53 AM

**Comment by wanda-carlson** [comment]
This is an image...
Screen Shot 2022-01-10 at 10 25 58 AM

**Comment by wanda-carlson** [comment]
Readme file here
[README.md](https://github.com/PaidPaper/public/files/7965418/README.md)

**Comment by wanda-carlson** [comment]
This is another image
[Screen Shot 2022-01-06 at 7.59.26 AM.png](https://files.slack.com/files-pri/T0289BY1FK4-F0316B8EJGH/screen_shot_2022-01-06_at_7.59.26_am.png)

**Comment by wanda-carlson** [comment]
This is an image
Screen Shot 2022-01-24 at 9 16 53 AM

**Review by henry-richard — CHANGES_REQUESTED** [review]
Make some changes

**Review by henry-richard — APPROVED** [review]
Looking good!

**Comment by henry-richard** [comment]
```Some pictures and information```
Screen Shot 2022-01-27 at 9 57 21 PM

**Review by henry-richard — APPROVED** [review]
Nice WOrk!

**Comment by wanda-carlson** [comment]
Can i merge this? **Henry Richard**

**Comment by henry-richard** [comment]
Go ahead

**Pull request closed without merging**